sdv/constraints/tabular.py

```diff
@@ -230,6 +237,9 @@ def _fit(self, table_data):
                 The Table data.
         """
         self._dtype = table_data[self._high].dtype
+        self._diff_column = f'#{self._low}#{self._high}'
```

Comment:
Maybe we should be [finding a valid separator in a similar fashion as to what `UniqueConstraints` does](https://github.com/sdv-dev/SDV/blob/1dbbfdaca61b532ca85c9ddedde5c296b0cbccc7/sdv/constraints/tabular.py#L109)? We could move the logic to a `_find_valid_separator(table_data, columns` staticmethod at the `Constraint` class (or even `sdv.utils`), so all the constraints (and potentially other parts of SDV) can use it.
I would do either that or involve a `uuid` generated on the fly to make sure that the column name does not clash with an existing column.